Source/Core/DolphinQt/FIFO/FIFOPlayerWindow.cpp

```diff
@@ -297,14 +297,14 @@ void FIFOPlayerWindow::OnFIFOLoaded()
 {
   FifoDataFile* file = FifoPlayer::GetInstance().GetFile();
 
-  auto object_count = FifoPlayer::GetInstance().GetFrameObjectCount();
+  auto object_count = FifoPlayer::GetInstance().GetMaxObjectCount();
   auto frame_count = file->GetFrameCount();
 
   m_frame_range_to->setMaximum(frame_count);
-  m_object_range_to->setMaximum(object_count);
+  m_object_range_to->setMaximum(object_count - 1);
 
   m_frame_range_to->setValue(frame_count);
-  m_object_range_to->setValue(object_count);
+  m_object_range_to->setValue(object_count - 1);
```

Comment:
These (/ `FIFOPlayerWindow::OnFIFOLoaded()` in general) are also called when the FIFO is unpaused, which turns out to be rather annoying since it changes the upper bound. (Also a problem with 5.0-13603)

(moved from [here](https://github.com/dolphin-emu/dolphin/pull/9497#discussion_r581567296))